classes/log/store.php

```diff
@@ -108,7 +112,7 @@ public function process_events(array $events) {
                 'lrs_endpoint' => $this->get_config('endpoint', ''),
                 'lrs_username' => $this->get_config('username', ''),
                 'lrs_password' => $this->get_config('password', ''),
-                'lrs_max_batch_size' => $this->get_config('maxbatchsize', 100),
+                'lrs_max_batch_size' => $this->get_max_batch_size(),
```

Comment:
Real quick, do these event's get processed from oldest to newest or newest to oldest? Because if it's newest to oldest, the batch size could be small enough, that we start to get a backlog in bigger systems, under heavier usage. So we end up with events, never getting sent, because the amount of new events is greater then the batch size per cron run. Especially with systems looking seeing 1k users worth of traffic. Just a curious question.

Reply:
Looking at the code, I believe it would be oldest to newest. I'm going to merge this and we can resolve that issue separately when needed.